config/config.json

```diff
@@ -1,7 +1,8 @@
 {
     "prefix": "??",
     "considerationPeriod": 15000,
-    "topNEntries": 5,
+    "maxEntriesLimit": 15,
+    "defaultEntriesLimit": 5,
```

Comment:
Through development of this feature, 5 as a default seems a bit limited. Maybe 10? What do you think?

Reply:
Sounds good.